src/screencast_keys/ops.py

```diff
@@ -1371,15 +1382,31 @@ def event_timer_remove(cls, context):
                 wm.event_timer_remove(cls.timers[key])
         cls.timers.clear()
 
-    def invoke(self, context, event):
-        cls = self.__class__
-        prefs = compat.get_user_preferences(context).addons[__package__].preferences
+    @classmethod
+    def start(cls, self, context, event, prefs, is_start):
+        if is_start:
+            if is_running():
+                return
```

Comment:
I think is the impossible case, so return error code and check the value in the caller functions.

Reply:
It's a preventive way to not run unexpected code. But if we have not found any problems, I should remove it.